src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -25,18 +34,80 @@ private void initMenu() {
         // Add menus with gaps
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
 
-        this.add(this.importMenu);
+        this.add(this.fileMenu);
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
-        // importMenu should be disabled  before signing in
-        this.importMenu.setEnabled(false);
-
         this.add(this.settingsMenu);
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
-
         this.add(this.helpMenu);
+
+        // Add MenuItem import in File
+        JMenuItem importMenuItem = new JMenuItem("Import File");
+        fileMenu.add(importMenuItem);
+
+        // Add function to import menu
+        importMenuItem.addActionListener( e -> {
+            importBills();
+        });
+        this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
+
+        // Add MenuItem change color and font in setting
+        JMenuItem appearanceMenuItem = new JMenuItem("Appearance");
+        settingsMenu.add(appearanceMenuItem);
+
+        // Add function to setting menu
+        appearanceMenuItem.addActionListener( e -> {
+            System.out.println("yes");
+            int[] setting = setting();
+
+            mainFrame.getBillPanel().changeColor(COLORS[setting[0]]);
+            mainFrame.getActionPanel().changeColor(COLORS[setting[0]]);
+
+            mainFrame.getActionPanel().changeFont(FONTS[setting[1]]);
+            mainFrame.getBillPanel().changeFont(FONTS[setting[1]]);
+
+        });
+    }
+
+    // Function to set up the setting dialog
+    private int[] setting() {
+//        SettingDialog setdlg = mainFrame.getSettingDialog();
+        SettingDialog setdlg = new SettingDialog(mainFrame);
+        int[] setting = new int[2];
+        if (setdlg.exec()) {
+            int colorTargetIndex = setdlg.getMyColor();
+            int fontTargetIndex = setdlg.getMyFont();
+            SettingDialog.setFontField(fontTargetIndex);
+            SettingDialog.setColorField(colorTargetIndex);
+            setting[0] = colorTargetIndex;
+            setting[1] = fontTargetIndex;
+            return setting;
+        }
+        setting[0] = settingDialog.getMyColor();
+        setting[1] = settingDialog.getMyFont();
```

Comment:
These two lines cause a `NullPointerException` when you click the cancel button.
This is because your `settingDialog` does not initialize.
The simple solution is change `settingDialog` to `setdlg`.

Also, remember to remove the `settingDialog` instance variable declaration: `settingDialog` are useless (at least I don't see anywhere you used it).